rewrite-groovy/src/test/java/org/openrewrite/groovy/tree/LiteralTest.java

```diff
@@ -40,9 +40,46 @@ void insideParentheses() {
     }
 
     @Test
-    void string() {
+    void singleQuoteString() {
         rewriteRun(
-          groovy("'hello'")
+          groovy(
```

Comment:
please add `//language=groovy` to enable hightlights for everyone

Reply:
There should not be a need, because we use it in our source code:

```java
public static SourceSpecs groovy(@Language("groovy") @Nullable String before) {
 ..
}
```

Sometimes the hightlighing does work do and we also use `//language=groovy`. I think it's an IntelliJ bug if it does not work, for [both](https://www.jetbrains.com/help/idea/using-language-injections.html#use-language-injection-comments) options are supported. As we don't use `//language=groovy` in the rewrite-groovy module, I don't want to add it here.